app/src/main/res/layout/activity_calculator.xml

```diff
@@ -33,20 +33,21 @@
 
         <com.google.android.material.textfield.TextInputEditText
             android:id="@+id/tietParamB"
-            android:inputType="numberDecimal"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:hint="hint" />
+            android:hint="hint"
+            android:inputType="numberDecimal" />
     </com.google.android.material.textfield.TextInputLayout>
 
     <Button
-        android:id="@+id/btnPerformOperation"
+        android:id="@+id/btnPerformOperationAddiction"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginBottom="8dp"
-        android:text="Button"
+        android:text="+"
```

Comment:
Maybe doing some internationalization here can be good.

Reply:
resolved

Reply:
would it be necessary to internationalize buttons with arithmetic operations?

Reply:
resolved